src/FsAutoComplete/LspHelpers.fs

```diff
@@ -73,8 +73,8 @@ module Conversions =
             {
                 Uri = fileUri
                 Range = {
-                    Start = { Line = ex.Line - 1; Character = ex.Column - 1 }
-                    End = { Line = ex.Line - 1; Character = ex.Column - 1 }
+                    Start = { Line = ex.Line - 1; Character = ex.Column }
```

Comment:
like the above change, these quasi-positions from the compiler no longer need to be down-shifted in order to be translated to LSP positions